.circleci/config.yml

```diff
@@ -651,7 +653,7 @@ workflows:
       - devhub
       - main
       # Uncomment if you want to test the docker build
```

Comment:
I assume you are you going to re-comment the job before merging ?

Reply:
I actually would rather leave it for now. The next few PRs will benefit from comparison against Circle build, but happy to re=comment and un-comment in each PR if you prefer.

Reply:
Re-commented, will just enable when needed.